frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -67,18 +69,12 @@ const TranslationPage = () => {
     setFontSize(val);
   };
 
-  const deepCopy = (lines: Object) => {
-    // This is a funky method to make deep copies on objects with primative values
-    // https://javascript.plainenglish.io/how-to-deep-copy-objects-and-arrays-in-javascript-7c911359b089
-    // Should probably go under some util
-    return JSON.parse(JSON.stringify(lines));
-  };
-
   const onChangeTranslationContent = async (
```

Comment:
how come this is marked as async? it doesn't look like the async is necessary in this function

Reply:
good catch! does not have to be async. just tested it out too.